src/main/java/at/ac/fhcampuswien/fhmdb/controller/AbstractViewController.java

```diff
@@ -0,0 +1,145 @@
+package at.ac.fhcampuswien.fhmdb.controller;
+
+import at.ac.fhcampuswien.fhmdb.dao.WatchlistRepository;
+import at.ac.fhcampuswien.fhmdb.exceptions.DatabaseException;
+import at.ac.fhcampuswien.fhmdb.handler.ClickEventHandler;
+import at.ac.fhcampuswien.fhmdb.models.Movie;
+import at.ac.fhcampuswien.fhmdb.ui.MovieCell;
+import at.ac.fhcampuswien.fhmdb.ui.SVG;
+import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXListView;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonBar;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.SVGPath;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+
+public abstract class AbstractViewController {
+
+    @FXML
+    protected JFXListView<Movie> movieListView = new JFXListView<>();
+
+    @FXML
+    protected VBox navigationMenu;
+
+    @FXML
+    public JFXButton navigationButton;
+
+    @FXML
+    public JFXButton aboutButton;
+
+    protected SVGPath cross = new SVGPath();
+
+    protected SVGPath burger = new SVGPath();
+    protected final ObservableList<Movie> movies = FXCollections.observableArrayList();
+
+    protected ClickEventHandler<Movie> onWatchlistButtonClicked;
+
+    protected WatchlistRepository repository;
+
+    public static final String NO_DB_CONNECTION_AVAILABLE = "No database connection available";
+
+    public void initialize() {
+
+        if (repository == null) {
+            try {
+                repository = new WatchlistRepository();
+            } catch (DatabaseException e) {
+                showAlertMessage(e.getMessage());
+            }
+        }
+
+        movies.addAll(getAllMoviesOrEmptyList());
+        movies.sort(Comparator.naturalOrder());
+
+        burger.setContent(SVG.BURGER);
+        burger.setStroke(Color.WHITE);
+        burger.setStrokeWidth(4);
+
+        cross.setContent(SVG.CROSS);
+        cross.setStroke(Color.WHITE);
+        cross.setStrokeWidth(4);
+
+        navigationButton.setGraphic(burger);
+
+        movieListView.setItems(movies);
+        movieListView.setCellFactory(e -> new MovieCell(onWatchlistButtonClicked));
```

Comment:
### Remove Button not shown in the Movie-Cell

Refactor to this
`movieListView.setCellFactory(e -> new MovieCell(onWatchlistButtonClicked, isWatchlistCell));`